examples/basic/android/gradle.properties

```diff
@@ -18,3 +18,5 @@
 # org.gradle.parallel=true
 
 android.useDeprecatedNdk=true
```

Comment:
I believe android.useDeprecatedNdk is no longer supported?

Reply:
should I remove the property then?